docs/content/docs/start/tutorial.md

```diff
@@ -7,188 +7,158 @@ aliases:
   - "/start/tutorial/"
 ---
 
-In this walkthrough, you'll learn about using the Fluid Framework by building a simple [DiceRoller](https://github.com/microsoft/FluidHelloWorld) application. To get started, go
-through the [Quick Start]({{< relref "./quick-start.md" >}}) guide.
+In this walkthrough, you'll learn about using the Fluid Framework by building a simple [DiceRoller](https://github.com/microsoft/FluidHelloWorld) application. To get started, go through the [Quick Start]({{< relref "quick-start.md" >}}) guide.
```

Comment:
I feel like in general the doc can use more guidance relative to the actual repo. Some Examples:

- After you run the Quick Start launch the root folder with an IDE.
- In this following section we will be looking at `./src/app.ts`
- In the following section we will be looking at `./src/view/jsView.ts`

Reply:
good point. I haven't really connected it to the repo code at all, partly because it hasn't been updated yet :) But I'll make some changes to fix that.

Reply:
```suggestion
In this walkthrough, you'll learn about using the Fluid Framework by examining the DiceRoller application at [https://github.com/microsoft/FluidHelloWorld](https://github.com/microsoft/FluidHelloWorld). To get started, go through the [Quick Start]({{< relref "quick-start.md" >}}) guide.
```